admob/src/ios/load_ad_result_ios.mm

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "admob/src/ios/load_ad_result_ios.h"
+
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+#include "admob/src/include/firebase/admob.h"
+#include "admob/src/ios/response_info_ios.h"
+#include "app/src/util_ios.h"
+
+namespace firebase {
+namespace admob {
+
+LoadAdResult::LoadAdResult() : AdResult(), response_info_() {}
+
+LoadAdResult::LoadAdResult(const LoadAdResultInternal& load_ad_result_internal)
```

Comment:
Let me make sure I've got this clear, because it seems odd to me...

- AdResult contains an AdResultInternal - this is similar to the model we use for the other modern SDKs.
- LoadAdResult is a subclass of AdResult, and does *not* contain a LoadAdResultInternal.
  - Instead, LoadAdResultInternal contains an AdResultInternal
  - When a LoadAdResult is created, the LoadAdResultInternal's AdResultInternal is set in the base AdResult class.

Does that sound like what's happening? If so, LoadAdResultInternal seems weird and extraneous, no?

Reply:
Yes, you're right.  It once had more information but I have since removed the other fields.  I'll move `LoadAdResultInternal` and just use `AdResultInternal` in all cases.

Reply:
Done.